docs/release-notes/list-active-users.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**New Features**
+
+-  CLI: ``det users ls`` now only shows active users. New flag ``--all`` added which will show all users.
```

Comment:
:orphan:

**New Features**

-  CLI: Modify ``det users ls`` to show only active users. Add a new flag ``--all`` to show all users.